src/schema.js

```diff
@@ -52,8 +52,7 @@ const signinSchema = Joi.object({
 
 const createArticleSchema = Joi.object({
 	body:{
-		userId: Joi.number()
-			.required(),
+		userId: Joi.number(),
```

Comment:
@itzgeebee this is slightly different from what we had

to get the original file before the change you could simply run

```
git checkout origin/main src/schema.js
```
and commit and push it

Reply:
fixed it. Thanks @jocrah